accord-core/src/main/java/accord/primitives/Keys.java

```diff
@@ -180,6 +181,48 @@ public static <V> Keys of(List<V> input, Function<? super V, ? extends Key> tran
         return of(array);
     }
 
+    public static <A, B> Keys ofMergeSorted(List<A> as, Function<? super A, ? extends Key> fa, List<B> bs, Function<? super B, ? extends Key> fb)
```

Comment:
This is unused in this PR, so I assume it will be used in a later one.

Reply:
I believe this is used in integration to respond to one of your other pieces of feedback